### PR TITLE
Fix popularity community

### PR DESCRIPTION
**Pull request by xoriole**
- Fixes a bug on that popularity community was not receiving torrent checker instance because of which no popular torrents are shared in the community.
- Now the popularity community only shares alive torrents

### CODE REVIEW

File: src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py [hunk 1]
```diff
@@ -209,6 +210,8 @@ def update_torrents_checked(self, new_result):
         new_result_tuple = (new_result['infohash'], new_result['seeders'],
                             new_result['leechers'], new_result['last_check'])
         self.torrents_checked.add(new_result_tuple)
+        if new_result['seeders'] > 0:
+            self.torrents_checked.add(new_result_tuple)
```
Comment (devos50) [inline src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py:214]: You are now adding the `new_result_tuple` twice. Is this intended?

Comment (xoriole) [reply src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py:214]: Good point! Thank you for pointing it out

**Review by qstokkink — DISMISSED** [review]

**Review by qstokkink — DISMISSED** [review]

**Comment by devos50** [comment]
Shouldn't this go in `release-7.5` ?

**Comment by xoriole** [comment]
retest this please

**Comment by devos50** [comment]
Sonarcloud might be confused since the base branch has been changed from GitHub itself?

**Comment by xoriole** [comment]
Yes, seems like it. I'll rebase and push again.

**Review by devos50 — APPROVED** [review]

**Pull request merged**